lametro/templates/lametro/legislation.html

```diff
@@ -97,20 +98,19 @@ <h3 class="no-pad-bottom"><i class='fa fa-fw fa-list-ul'></i> History</h3>
                             </tr>
                         </thead>
                         <tbody>
-                            {% for action in actions %}
+                            {% for action in events %}
                                 <tr>
                                     <td class='nowrap text-muted small'>
-                                        <span datetime='{{action.date_dt | format_date_sort }}'>{{action.date_dt|date:'n/d/y'}}</span>
+                                        <span datetime='{{action.date}}'>{{action.date}}</span>
                                     </td>
                                     <td class="small">
-                                        <span class='text-{{action.label}}'>{{action.description | remove_action_subj}}</span>
-                                        {% if action.related_organization %}
-                                            to
-                                            {{action.related_organization.link_html|safe}}
-                                        {% endif %}
+                                        <span class='text-default'>{{action.description}}</span>
                                     </td>
                                     <td class="small">
-                                        {{action.organization.link_html|safe}}
+                                            {{ action.organization.link_html|safe }}
```

Comment:
Shelly asked that we link to the meeting, rather than the organization. Luckily, people, organizations, and events all have [a `link_html` property](https://github.com/datamade/django-councilmatic/blob/ce203941540ccf70d4671823b49b254c3b4d35b4/councilmatic_core/models.py#L348), so you just need to change the reference here.

```suggestion
                                            {{ action.event.link_html|safe }}
```

Reply:
@hancush would it be appropriate to change the name of the column here as well?

Reply:
Yes, great call (and good catch)! One of the original requests was to change the column heading from Committee to Meeting: https://github.com/datamade/la-metro-councilmatic/issues/348#issuecomment-668258972